robottelo/performance/perf_stat.py

```diff
@@ -0,0 +1,41 @@
+"""Test utilities for writing csv files"""
+import csv
+import numpy as np
```

Comment:
Please use numpy instead of np, that will be better when other are reading. Also as you have added a new requirement, it should be listed on the setup.py https://github.com/SatelliteQE/robottelo/blob/master/setup.py#L5